winui/ComboBox/Highlighting.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title: Highlighting Feature in WinUI Combobox control | Syncfusion
+description: Learn here all about Highlighting Feature in Syncfusion WinUI Combobox control into WinUI application.
+platform: winui
+control: SfComboBox
+documentation: UG
+---
+
+# Highlighting in WinUI ComboBox(SfComboBox)
+
+Highlight matching characters in a suggestion list to pick an item with more clarity. There are two ways to highlight the matching text:
+
+* **Highlight beginning text**
+* **Highlight all occurance of search text**
+
+
+The text highlight can be indicated with various customizing styles by enabling the below properties. They are
+
+* **HighlightedTextColor** - sets the color of the highlighted text for differentiating the highlighted characters.
+
+* **HighlightedTextFontStyle** - sets the font style of the highlighted text.
+        
+* **HighlightedTextFontWeight** - sets the font weight of the highlighted text.
+
+* **HightlightedTextFontSize** - sets the font size of the highlighted text.
+
+{% tabs %}
+{% highlight XAML %}
+    <editors:SfComboBox 
+                            x:Name="comboBox"
+                            DisplayMemberPath="Name"
+                            IsEditable="true"
+                            IsFilteringEnabled="True"                       
+                            TextHighlightMode="Matched"                                
+                            TextSearchMode="StartsWith"                               
+                            HighlightedTextFontSize="15"                               
+                            HighlightedTextFontStyle="Italic"                               
+                            HighlightedTextFontWeight="Medium"                                
+                            HighlightedTextForeground="Red"                                
+                            ItemsSource="{Binding SocialMedias}">       
+    </editors:SfComboBox>
+
+{% endhighlight %}
+{% highlight C# %}
+        comboBox.DisplayMemberPath = "Name";
+        comboBox.IsEnabled=true;
+        comboBox.IsFilteringEnabled = true;          
+        comboBox.TextHighlightMode = ComboBoxTextHighlightMode.Unmatched;
+        comboBox.TextSearchMode = ComboBoxTextSearchMode.Contains;
+        comboBox.HighlightedTextFontSize = 10;
+        comboBox.HighlightedTextFontStyle = FontStyle.Normal;
+        comboBox.HighlightedTextFontWeight = FontWeights.ExtraLight;
+        comboBox.HighlightedTextForeground = Color.Red;
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI ComboBox text highlighting properties](Highlighting_images/winui-combobox-highlightingproperties.png)
+
+
+
+### **Highlight beginning text**
+It highlights the matches that starts with the typed characters in suggestion list.
```

Comment:
It highlights the matches that start with the typed characters in the suggestion list.